L1Trigger/L1TGlobal/plugins/L1TGlobalProducer.cc

```diff
@@ -556,6 +565,11 @@ void L1TGlobalProducer::produce(edm::Event& iEvent, const edm::EventSetup& evSet
   // * produce the GlobalObjectMapRecord
   std::unique_ptr<GlobalObjectMapRecord> gtObjectMapRecord(new GlobalObjectMapRecord());
 
+  std::unique_ptr<AXOL1TLScoreBxCollection> uGtAXOScoreRecord(nullptr);
+  if (m_saveAxoScore) {
```

Comment:
In the L1TGlobalProducer, it should be  ```if (m_produceAXOL1TLScore) {```.

Reply:
good catch, Ill fix this - thanks